ui/responsive/app/app.js

```diff
@@ -125,7 +123,7 @@ App.prototype.renderAppBar = function () {
           alignItems: 'center',
           visibility: props.isUnlocked ? 'visible' : 'none',
           background: props.isUnlocked ? 'white' : 'none',
-          height: '36px',
+          height: '38px',
```

Comment:
We felt the extra 2px made sense for now, given the contrast between colors. 
![](https://puu.sh/wMFh7/85e6042fd9.png)